source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst

```diff
@@ -55,3 +55,25 @@ is supported on platforms:
 		root@am62xx-evm:~# modprobe ili210x
 
 After re-initializing the module, the touch IC should now be registered as an input and touch should be functional.
+
+**Note**: The following steps are only applicable on AM625 SK EVM revision B/B-P1 board
```

Comment:
Better to link

https://www.ti.com/tool/SK-AM62B

https://www.ti.com/tool/SK-AM62B-P1


Similar to how @praneethbajjuri mentions in his PR - https://github.com/TexasInstruments/processor-sdk-doc/pull/77/files#diff-b18514cfd2d8f65bf16517bf5593a255d5417eb1594fe4e1157fd93291f78e06

Reply:
Done